internal/interfaces/grpc/permissions/permissions_test.go

```diff
@@ -34,6 +36,10 @@ func TestWhitelistedMethods(t *testing.T) {
 		allMethods = append(allMethods, fmt.Sprintf("/%s/%s", tdexv2.TradeService_ServiceDesc.ServiceName, v.MethodName))
 	}
 
+	for range grpchealth.Health_ServiceDesc.ServiceName {
+		allMethods = append(allMethods, fmt.Sprintf("/%s/%s", grpchealth.Health_ServiceDesc.ServiceName, "Check"))
+	}
```

Comment:
```suggestion
	for _, v := range grpchealth.Health_ServiceDesc.ServiceName {
		allMethods = append(allMethods, fmt.Sprintf("/%s/%s", grpchealth.Health_ServiceDesc.ServiceName, v.MethodName))
	}
```

This implies adding Watch to the whitelist even if it's not implemented basically

Reply:
Is it better if we dont add it to whitelist cause it is not implemented?

Reply:
alright, so this should change like: 
```suggestion
	allMethods = append(allMethods, fmt.Sprintf("/%s/%s", grpchealth.Health_ServiceDesc.ServiceName, "Check"))
```